Fix small issues and mistyping (#285)
sora-xor/sora-docs: src/tbcd.md

# Token Bonding Curve Dollar (TBCD)

> - Token Bonding Curve Dollars (TBCD) is a proposed, algorithmic, non-synthetic stablecoin that has its value maintained by the SORA token bonding curve.

Reviewer: ```suggestion
- Token Bonding Curve Dollars (TBCD) is an algorithmic, non-synthetic stablecoin whose value is maintained by the SORA token bonding curve.
```

- TBCD is created to be worth slightly less than $1 when XOR is far lower than the price point on the token bonding curve, reducing sell pressure to XOR when the price of XOR is low.

> - TBCD will help to build up reserves in the SORA token bonding curve, while also being used to fund the creation of new goods and services via an on-chain referendum decided by all XOR token holders.

Reviewer: ```suggestion
- TBCD helped build up reserves in the SORA token bonding curve, while also used to fund the creation of new goods and services via an on-chain referendum decided by all XOR token holders.
```

## Algorithmic Central Banking

> TBCD is a great example showcasing the possibilities of the SORA token bonding curve to create monetary innovation for the SORA economy. TBCD solves both the question of how to fund the SORA economy’s productive expansion (the creation of new goods and services) and how to build up reserves in the token bonding curve to create a backing aimed at increasing the stability of XOR. By minting TBCD via on-chain referendum, builders can get the resources they need to build up the SORA economy and platform, while any TBCD they sell to the SORA token bonding curve will be owned by the SORA ecosystem as protocol-owned liquidity.

Reviewer: ```suggestion
TBCD was a great example showcasing the possibilities of the SORA token bonding curve to create monetary innovation for the SORA economy. 
TBCD sought to solve both the question of how to fund the SORA economy’s productive expansion (the creation of new goods and services) and how to build up reserves in the token bonding curve to create a backing to increase the stability of XOR. By minting TBCD via on-chain referendum, builders could get the resources they need to build up the SORA economy and platform. At the same time, the SORA ecosystem owns any TBCD sold to the SORA token bonding curve as protocol-owned liquidity.
```
